examples/platforms/da15000/radio.c

```diff
@@ -413,7 +416,7 @@ void FTDF_sendFrameTransparentConfirm(void         *handle,
 void da15000RadioProcess(otInstance *aInstance)
 {
     bool rxPending;
-    uint8_t value = 0;
+    uint8_t bufferFor_FTDF_configValue = 0;
```

Comment:
I won't argue with you coding standards, but it should be a good practice to use underscore bevore abbreviations.